vite.config.ts

```diff
@@ -19,21 +19,24 @@ export default defineConfig({
     lib: {
       entry: resolve('src', 'index.ts'),
       name: pkg.name,
-      formats: ['es', 'cjs'],
     },
     rollupOptions: {
       external: [
-        'antd-5',
         'react',
         'react-dom',
       ],
-      output: {
-        globals: {
-          antd: 'antd-5',
-          react: 'React',
-          'react-dom': 'ReactDOM',
+      output: [
```

Comment:
globals configuration was used to avoid clashed with Ant versions. As an example, when a design system's component is used inside the website package, imports from ant5 will fail. This happens because Vite is not able to differentiate correctly between the two versions of Ant used (3 by website and 5 by design system). We mirrored, as a temporary solution, the package renaming convention already adopted inside website for Ant 5. We are aware that is not optimal though. 

Could you propose an alternative solution to avoid this configuration?

Reply:
This lib is already a bundle with only ant5. Are you sure there is the problem?
Because in the example, in one lib there was 2 different versions of ant